tutorials/quickstart/quickstart-tutorial-intro-paragraph.include.md

```diff
@@ -1 +1,5 @@
-Learn why software systems built on asynchronous messaging using NServiceBus are superior to traditional synchronous HTTP-based web services. We'll also show how NServiceBus guarantees reliability and extensibility that can't be achieved with REST.
+It is **very difficult** to build a distributed software system correctly from scratch. Would you rather read all 736 pages of the [Enterprise Integration Patterns](https://www.enterpriseintegrationpatterns.com/) book and then spent months creating, testing, and documenting a communication framework so that different services can talk to each other? Or would you rather use a framework that builds on well-defined distributed system design principles and guides you toward the "pit of success?"
+
+NServiceBus combines decades of distributed systems design experience and expertise, and distills it into one easy-to-use framework. In this tutorial, you'll see how NServiceBus takes all the gruntwork out of system design by handling all of the plumbing for you, taking system design best practices like reliability, failure recovery, and extensibility and baking them right in to the software, guiding you toward the pit of success.
```

Comment:
```suggestion
NServiceBus combines decades of distributed systems design experience and expertise, and distills it into one easy-to-use framework. In this tutorial, you'll see how NServiceBus takes all the gruntwork out of system design by handling all of the plumbing for you, taking system design best practices like reliability, failure recovery, and extensibility and baking them right into the software, guiding you toward the pit of success.
```

Duplicated "pit of success" here and in the paragraph above. I'd remove this one and keep the first.